src/gateways/help-request-call.js

```diff
@@ -1,7 +1,17 @@
 import { DefaultGateway } from '../gateways/default-gateway';
 
+const ToPostHelpRequestCall = (hr) => {
+  const callType = (hr.callType == 'CEV') ? 'Shielding': hr.callType
```

Comment:
I created a new constants file recently, can you move these two words (and other usages of it) to use that now please?

See 
https://github.com/LBHackney-IT/coronavirus-here-to-help-frontend/blob/7074b0269cb4a9a937716230153ce85963a94658/src/helpers/constants.js#L1


https://github.com/LBHackney-IT/coronavirus-here-to-help-frontend/blob/7074b0269cb4a9a937716230153ce85963a94658/src/pages/reassign-call/%5BrequestId%5D/index.js#L8
https://github.com/LBHackney-IT/coronavirus-here-to-help-frontend/blob/7074b0269cb4a9a937716230153ce85963a94658/src/pages/reassign-call/%5BrequestId%5D/index.js#L37

Reply:
nice, will do!